release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -553,6 +583,17 @@ In {op-system-base-full} 9, the default mode is cgroup v2. When {op-system-base-
 
 The Cluster Samples Operator is deprecated with the {product-title} 4.16 release. The Cluster Samples Operator will stop managing and providing support to the non-S2I samples (image streams and templates). No new templates, samples or non-Source-to-Image (Non-S2I) image streams will be added to the Cluster Samples Operator. However, the existing S2I builder image streams and templates will continue to receive updates until the Cluster Samples Operator is removed in a future release.
 
+[id="ocp-4-16-deprecation-operator-sdk_{context}"]
+==== Operator SDK and related Operators are deprecated
+
+The Operator SDK is deprecated with the {product-title} {product-version} release. Operator SDK is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. Additionally, the following related Operators are also deprecated and will be removed in a future release:
```

Comment:
`Additionally, the following related Operators are also deprecated and will be removed in a future release`

I don't think I would word this this way. We aren't actually deprecating any operators, we're deprecating the base image and scaffolding tools used to build operators that have them as a basis.

Maybe something like

`Additionally, the following related Operator base images are also deprecated and will be removed in a future release` ?

Reply:
Addressed.